### ISSUE TITLE
upgrade versions in-place?

### ISSUE THREAD

**Issue by Curbed-Enthusiasm** [vanderby/SonarQube-AzureAppService#52]
Per the title, is it possible to upgrade versions in place? Is it recommended? 

Currently we've been using this at 8.7, and don't see any issues upgrading. The solution has treated us well so far, we're just trying to keep things up-to-date where we can.

**Comment by Curbed-Enthusiasm** [comment]
sweet! thanks for the response! I'll give that a shot and report back in case anyone else finds it useful.

**Comment by thermick** [comment]
Hello Vnaderby, I thank you very much and I found it so useful, currently I have an App Service deployed with your template in a 8.6 community version and I am trying to update it to an enterprise version and the most current possible, but I do not understand much how to do it from the slot, I am doing what following and I do not know if you can help me:

- From "Deployment Slots"> "Add Slot"> Name: XXX> Clone settings from: "Prod Slot"
- I did the back up for the DB
- I cloned de setting and I understand that the connection string is the same, ok?:
    "name": "sonar.jdbc.url",
    "value": "jdbc:sqlserver://....."
- In the step "update the deployment slot with the new bits to test the upgrade"
   -   Do I need to copy the "site" folder from the production slot with the new bits and run HttpPlatformHandlerStartup.ps1?

I thank you in advance, attentive to your comments

**Comment by vanderby** [comment]
Hey @thermick, you can update the bits by downloading them from SQ's website then uploading them via FTP or App Service's Kudu backend or any other standard means of moving bits to an Azure App. Service.

**Comment by Curbed-Enthusiasm** [comment]
Just as a late follow-up because this upgrade was pushed back due to higher priority roadmap items. For testing I stood up a second instance using your repo, and backed up our prod db and hooked it up to the secondary instance then ran the upgrade steps via sonarqube's instructions and the FTPS settings in the app service. Unfortunately I just couldn't get the swap slot to work properly, but I just moved to an azure shop from an AWS one so I'm still learning some ins and outs. It was incredibly easy and worked like a charm, and the behavior was exactly the same when I ran everything on prod.

 Thanks again for putting all this together!
